bentoml/adapters/tensorflow_tensor_output.py

```diff
@@ -43,12 +43,15 @@ def tf_to_numpy(tensor):
 
 class TfTensorOutput(JsonOutput):
     """
-    Converts result of user defined API function into specific output.
+    Output adapters converts returns of user defined API function into specific output,
+    such as HTTP response, command line stdout or AWS Lambda event object.
 
     Args:
-        cors (str): The value of the Access-Control-Allow-Origin header set in the
-            AWS Lambda response object. Default is "*". If set to None,
-            the header will not be set.
```

Comment:
Any reason not to remove the doc string for the cors parameter from these adapter classes?

Reply:
To clarify that this option is deprecated before fully removed?